include/fiction/technology/charge_distribution_surface.hpp

```diff
@@ -69,7 +84,7 @@ class charge_distribution_surface : public Lyt
     {
         if (!Lyt::is_empty_cell(c) && cs != sidb_charge_state::NONE)
         {
-            strg->charge_coordinates.insert_or_assign(c,cs);
+            strg->charge_coordinates.insert_or_assign(c, cs);
         }
         else if (!Lyt::is_empty_cell(c) && cs == sidb_charge_state::NONE)
```

Comment:
It is not really necessary to check for `!Lyt::is_empty_cell(c)` twice. You could do one big outer `if` and then do another `if-else` inside. Would prevent evaluating the same condition two times.

Reply:
corrected

Reply:
Not yet :sweat_smile: